File: new/dart-maor/hok-method-label.dart
// חוט · hok-method-label — תווית שיטת הו"ק (בנק/אשראי/מזומן/אחר). חוזה: hok-method-label.contract.md
// המרה מ-JS (new/atoms/hok-method-label.mjs) — התנהגות זהה-לחלוטין למקור (חוק-4). אפס-import (dart-core בלבד).
String hokMethodLabel(String m) {
  if (m == 'bank') return 'הו"ק בנקאית';
  if (m == 'card') return 'אשראי בסליקה';
  if (m == 'cash') return 'מזומן חודשי';
  // JS `m || 'אחר'`: מחרוזת ריקה = falsy ⇒ 'אחר'; אחרת m עצמו (כלל-7: truthiness מפורש).
  return m.isEmpty ? 'אחר' : m;
}

File: new/atoms/hok-method-label.mjs
/** חוט · hok-method-label — קודם אוטומטית (אפיון-Golden). חוזה: hok-method-label.contract.md */
export function hokMethodLabel(m, T) {
    if (m === T.k1)
        return T.k2;
    if (m === T.k3)
        return T.k4;
    if (m === T.k5)
        return T.k6;
    return m || T.k7;
}
